gtk4/src/subclass/widget.rs

```diff
@@ -102,24 +518,335 @@ unsafe extern "C" fn widget_compute_expand<T: ObjectSubclass>(
     let instance = &*(ptr as *mut T::Instance);
     let imp = instance.get_impl();
     let wrap = from_glib_borrow(ptr);
-    let mut hexpand_p: bool = from_glib(*hexpand_ptr);
-    let mut vexpand_p: bool = from_glib(*vexpand_ptr);
+    let mut hexpand: bool = from_glib(*hexpand_ptr);
+    let mut vexpand: bool = from_glib(*vexpand_ptr);
 
-    imp.compute_expand(&wrap, &mut hexpand_p, &mut vexpand_p);
-    *hexpand_ptr = hexpand_p.to_glib();
-    *vexpand_ptr = vexpand_p.to_glib();
+    imp.compute_expand(&wrap, &mut hexpand, &mut vexpand);
+    *hexpand_ptr = hexpand.to_glib();
+    *vexpand_ptr = vexpand.to_glib();
+}
+
+unsafe extern "C" fn widget_contains<T: ObjectSubclass>(
+    ptr: *mut gtk_sys::GtkWidget,
+    x: f64,
+    y: f64,
+) -> glib_sys::gboolean
+where
+    T: WidgetImpl,
+{
+    let instance = &*(ptr as *mut T::Instance);
+    let imp = instance.get_impl();
+    let wrap = from_glib_borrow(ptr);
+
+    imp.contains(&wrap, x, y).to_glib()
 }
 
 unsafe extern "C" fn widget_direction_changed<T: ObjectSubclass>(
     ptr: *mut gtk_sys::GtkWidget,
-    directnptr: gtk_sys::GtkTextDirection,
+    direction_ptr: gtk_sys::GtkTextDirection,
+) where
+    T: WidgetImpl,
+{
+    let instance = &*(ptr as *mut T::Instance);
+    let imp = instance.get_impl();
+    let wrap = from_glib_borrow(ptr);
+    let direction_wrap = from_glib(direction_ptr);
+
+    imp.direction_changed(&wrap, direction_wrap)
+}
+
+unsafe extern "C" fn widget_focus<T: ObjectSubclass>(
+    ptr: *mut gtk_sys::GtkWidget,
+    direction_type_ptr: gtk_sys::GtkDirectionType,
+) -> glib_sys::gboolean
+where
+    T: WidgetImpl,
+{
+    let instance = &*(ptr as *mut T::Instance);
+    let imp = instance.get_impl();
+    let wrap = from_glib_borrow(ptr);
+    let direction_type = from_glib(direction_type_ptr);
+
+    imp.focus(&wrap, direction_type).to_glib()
+}
+
+unsafe extern "C" fn widget_get_request_mode<T: ObjectSubclass>(
+    ptr: *mut gtk_sys::GtkWidget,
+) -> gtk_sys::GtkSizeRequestMode
+where
+    T: WidgetImpl,
+{
+    let instance = &*(ptr as *mut T::Instance);
+    let imp = instance.get_impl();
+    let wrap = from_glib_borrow(ptr);
```

Comment:
same here and for other overrides, It doesn't make sense to name this wrap :)

Reply:
How would you call it? That's how it is everywhere else :)

Reply:
Ah no my bad. I confused the wrap function with the wrap property